packages/longhorn/longhorn-1.1/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
 commit: b7c0b7177f8bddcfc4b338c20ed46632973ba8e9
-version: 100.0.2
+version: 100.0.1
```

Comment:
This was reverted since no changes are being made in this release once this annotation was removed. This is why it was removed from the `release.yaml` as well.